x-pack/plugins/security_solution/public/flyout/entity_details/user_right/index.tsx

```diff
@@ -94,6 +96,19 @@ export const UserPanel = ({
     { onSuccess: refetchRiskScore }
   );
 
+  const { data } = useMisconfigurationPreview({
```

Comment:
Not essential but it might be nice to have this logic in a `hasMisconfigurationFindings` hook?

Reply:
regarding your first question
we are planning to add another tab after this PR that will show Vulnerabilities :)